.github/workflows/govuln.yaml

```diff
@@ -0,0 +1,26 @@
+name: Go Vulnerability Checker
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - linux-amd64-govuln-check
```

Comment:
Are you planning to add multiple targets? If not maybe simplify it

Reply:
I was planning to add the same comment, but it isn't a big deal, so I gave up adding the comment.

Reply:
@vivekpatani feel free to address this comment in a separate PR if you want.

Reply:
@serathius @ahrtr https://github.com/etcd-io/etcd/pull/14669 - addressed